core_lib/src/structure/copymultiplekeyframesdialog.h

```diff
@@ -0,0 +1,84 @@
+#ifndef COPYMULTIPLEKEYFRAMESDIALOG_H
+#define COPYMULTIPLEKEYFRAMESDIALOG_H
+
+#include <QDialog>
+#include "layermanager.h"
+#include "editor.h"
+
+namespace Ui {
+class CopyMultiplekeyframesDialog;
+}
+
+class CopyMultiplekeyframesDialog : public QDialog
+{
+    Q_OBJECT
+
+public:
+    explicit CopyMultiplekeyframesDialog(QWidget *parent = nullptr);
+    explicit CopyMultiplekeyframesDialog(LayerManager *lm, int startLoop = 1, int stopLoop = 2, QWidget *parent = nullptr);
+    ~CopyMultiplekeyframesDialog();
+    void init();
+    int getFirstFrame();        // first frame in range
```

Comment:
FWIW you might have noticed doxygen supports several comment styles but IIRC we haven’t settled on a particular style yet (says a lot about the state of our documentation…). Just in case you were wondering.